Framework/Crystal/inc/MantidCrystal/SCDCalibratePanels.h

```diff
@@ -7,6 +7,7 @@
 #pragma once
 
 #include "MantidAPI/Algorithm.h"
+#include "MantidCrystal/DllConfig.h"
```

Comment:
I've noticed a couple of these there `DllConfig.h` is added but nothing else is changed, is this intentional?

Reply:
No that shouldn't be there. I'll remove